src/api.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```

Comment:
- [ ] `Linters` are there there to help us stick to best practices, please it will be nice to fix the issue rather than disabling `eslint`.

Reply:
fixed